pkgs/fc/lldp-to-altname/fc-lldp-to-altname.py

```diff
@@ -17,6 +18,10 @@ def has_known_prefix(name):
     return False
 
 
+def quote_altname(name):
+    return re.sub(r"[^a-z0-9A-Z]+", "-", name)
```

Comment:
I find the name "quote" a bit confusing, because I do not see how replacing non-ASCII-alphanumeric characters with a `-` is related to what I understand as quoting.

But if that works as intended I'm fine with it nonetheless.